### PR TITLE
grafana/toolkit: styling support

### PR DESCRIPTION
**Pull request by dprokop**
Ref #17819 

- [ ] Add support for common conventions:
  - [x] Single css/scss file (used via import)
  - [x] Separate files for themes (used via loadPluginCss)
  - [x] Emotion (to verify)
- [x] Enable static files
- [x] Enable PostCSS
- [x] Verify SASS setup

### CODE REVIEW

File: packages/grafana-toolkit/src/config/webpack/loaders.ts [hunk 1]
```diff
@@ -90,3 +96,26 @@ export const getStyleLoaders = () => {
...
+      test: /\.(png|jpe?g|gif|svg)$/,
+      use: [
+        {
+          loader: 'file-loader',
+          options: {
+            outputPath: 'static',
+            name: '[name].[hash:8].[ext]',
+            // When css is extracted the public path of statics is handled by MiniCssExtractPlugin.loader (see getStyleLoaders above)
+            // Otherwise the css is loaded using style-loader and we need to be specific what's the full public path for a given plugin
+            publicPath: shouldExtractCss ? undefined : `/public/plugins/${pluginJson.id}/static`,
```
Comment (ryantxu) [inline packages/grafana-toolkit/src/config/webpack/loaders.ts]: This may have weird behavior when running from the non-root path.

**Review by ryantxu — APPROVED** [review]
Looks good -- I'm not sure about the absolute reference to /public since sometimes it is actually /{somethign}/public

File: packages/grafana-toolkit/src/config/webpack/loaders.ts (hunk 1 above)
Comment (dprokop) [reply packages/grafana-toolkit/src/config/webpack/loaders.ts]: yeah, still trying to figure out the right approach here.

Comment (dprokop) [reply packages/grafana-toolkit/src/config/webpack/loaders.ts]: OK, changed the behaviour for now. The files are now inlined as base64 URIs (https://github.com/webpack-contrib/url-loader). Couldn't find an obvious/short path to solve that. The 'root_url' config option is not an issue I think as we can still have an absolute path with `public` (base tag is added to head). More problematic is `static_root_path` config option, which is know on runtime only.

**Pull request merged**